docs/architecture/modern-web-apps-azure/common-client-side-web-technologies.md

```diff
@@ -154,23 +154,33 @@ This is enough to render "Hello Vue!" on the page. Note, however, that Vue isn't
 
 Of course, this only scratches the surface of what Vue is capable of. It's gained a great deal of popularity in the last several years and has a large community. There's a [huge and growing list of supporting components and libraries](https://github.com/vuejs/awesome-vue#redux) that work with Vue to extend it as well. If you're looking to add client-side behavior to your web application or considering building a full SPA, Vue is worth investigating.
 
+### Blazor WebAssembly
+
+Unlike other JavaScript frameworks, `Blazor WebAssembly` is a single-page app (SPA) framework for building interactive client-side web apps with .NET. Blazor WebAssembly uses open web standards without plugins or recompiling code into other languages. Blazor WebAssembly works in all modern web browsers, including mobile browsers.
+
+Running .NET code inside web browsers is made possible by WebAssembly (abbreviated `wasm`). WebAssembly is a compact bytecode format optimized for fast download and maximum execution speed. WebAssembly is an open web standard and supported in web browsers without plugins.
+
+WebAssembly code can access the full functionality of the browser via JavaScript, called JavaScript interoperability, often shortened to JavaScript interop or JS interop. .NET code executed via WebAssembly in the browser runs in the browser's JavaScript sandbox with the protections that the sandbox provides against malicious actions on the client machine.
+
+For more details, see [Introduction to ASP.NET Core Blazor](https://docs.microsoft.com/aspnet/core/blazor/?view=aspnetcore-5.0)
+
 ### Choosing a SPA Framework
 
-When considering which JavaScript framework will work best to support your SPA, keep in mind the following considerations:
+When considering which option will work best to support your SPA, keep in mind the following considerations:
 
 - Is your team familiar with the framework and its dependencies (including TypeScript in some cases)?
 
 - How opinionated is the framework, and do you agree with its default way of doing things?
 
-- Does it (or a companion library) include all of the features your app requires?
+- Does it (or a companion library) includes all of the features your app requires?
```

Comment:
I wouldn't make this change. "Does it includes all of the features" doesn't make sense to me.

Reply:
I agree with @ardalis here, I'd leave it as "include"
```suggestion
- Does it (or a companion library) include all of the features your app requires?
```